### ISSUE TITLE
Indicate file status to the user since it can take a while for the build to start

### ISSUE THREAD

**Issue by eharris369** [eclipse/codewind#2022]
I create a go project from the Codewind go template on a remote connection (Open Shift). If I change the main.go file it takes about 10s after saving my changes before the build status changes to show that the file change was picked up. It would be nice if we could show the user that Codewind is busy copying the file changes over so they don't think that the file change was ignored.

**Comment by tobespc** [comment]
If the status is not updating soon enough then letting the user know something is happening is reasonable.  Not sure we would want to expose internal steps though.

**Comment by tetchel** [comment]
I like the idea of a 'syncing' or 'uploading' state before the build triggers

**Comment by tetchel** [comment]
You can see the need for this pretty obviously by creating multiple Microprofile projects at once (since they are large, require a lot of syncing). The projects get stuck in Disabled state for some time while they do the initial sync, and it would be worse on a slow connection. In the screenshot, `second` has been disabled for over two minutes.

![image](https://user-images.githubusercontent.com/8888438/76865433-1603e500-6839-11ea-9455-087aee0eb5bd.png)

**Comment by tetchel** [comment]
I still think this is one of the most important missing pieces in the auto-build use case. for example:

```
April 24, 12:24:36.892] [642.773] Received event [MODIFY] /Users/tim/vscode-codewind-workspace/⼬⼬⼬/Dockerfile
[April 24, 12:24:37.895] [643.776] Batch change summary for db4b4f80-8646-11ea-8f05-6d173efb5f8d @ 1587745476892: [ >Dockerfile ]
[April 24, 12:24:37.895] [643.776] Calling cwctl project sync with: { [--insecure] [project] [sync] [-p] [/Users/tim/vscode-codewind-workspace/⼬⼬⼬] [-i] [db4b4f80-8646-11ea-8f05-6d173efb5f8d] [-t] [1587745371867] }
[April 24, 12:24:50.169] [656.050] Successfully ran installer command: [--insecure] [project] [sync] [-p] [/Users/tim/vscode-codewind-workspace/⼬⼬⼬] [-i] [db4b4f80-8646-11ea-8f05-6d173efb5f8d] [-t] [1587745371867]
[April 24, 12:24:50.170] [656.051] Output:Status: 200 OK

[April 24, 12:24:50.170] [656.051] Updating timestamp to latest: 1587745477895
```

There's a 13 second period here (seconds 37-50) where there is no user feedback. And that's for syncing a 4KB file (why is it so slow)? A larger file means a much longer period where the user's source file edit appears to not have been picked up, unless you happen to be tailing the file-watcher logs.

**Comment by tobespc** [comment]
I agree @tetchel , these time periods where seemingly nothing is happening can be confusing. Will take a look to see if there is anything that can be done easily
